edx_sysadmin/api/test_views.py

```diff
@@ -28,59 +27,61 @@ def setUp(self):
         self.client = APIClient()
 
     @override_settings(SYSADMIN_GITHUB_WEBHOOK_KEY=SYSADMIN_GITHUB_WEBHOOK_KEY)
+    @override_settings(SYSADMIN_DEFAULT_BRANCH="master")
     @patch(
-        "edx_sysadmin.api.views.get_local_active_branch", return_value="ref/head/master"
+        "edx_sysadmin.api.views.get_local_active_branch",
+        return_value="refs/heads/master",
     )
     @patch("edx_sysadmin.api.views.get_local_course_repo", return_value=Repo())
     @patch("edx_sysadmin.api.views.add_repo")
     @ddt.data(
         (
-            "dd930da0a34996332e8c983aaeeb9e1cca45cc9b92492f47774d351de4740bb2",
-            "ref/head/master",
+            "d3a2424a1ad48d8441712400fd75392d56707a7b3e1dc4869239d87ee381cfa9",
+            "refs/heads/master",
             "push",
             "edx4edx_lite",
             "git@github.com:edx/edx4edx_lite.git",
-            status.HTTP_200_OK,
+            _status.HTTP_200_OK,
         ),
         (
             "dd930da0a34996332e8c983aaeeb9e1cca45cc9b92492f47774d351de4740ddc",
-            "ref/head/master",
+            "refs/heads/master",
             "push",
             "edx4edx_lite",
             "git@github.com:edx/edx4edx_lite.git",
-            status.HTTP_403_FORBIDDEN,
+            _status.HTTP_403_FORBIDDEN,
         ),
         (
-            "66228e6cbb54b81872b32325d8ee11ee9bca82605a89cf996dab1575f65d1413",
-            "ref/head/dev",
+            "98f48c1b0d35ceec2bfc420f96b76e500b538a6484b17c83c5df4d2c556d8c86",
+            "refs/heads/dev",
             "push",
             "edx4edx_lite",
             "git@github.com:edx/edx4edx_lite.git",
-            status.HTTP_400_BAD_REQUEST,
+            _status.HTTP_400_BAD_REQUEST,
         ),
         (
-            "dd930da0a34996332e8c983aaeeb9e1cca45cc9b92492f47774d351de4740bb2",
-            "ref/head/master",
+            "d3a2424a1ad48d8441712400fd75392d56707a7b3e1dc4869239d87ee381cfa9",
+            "refs/heads/master",
             "review",
             "edx4edx_lite",
             "git@github.com:edx/edx4edx_lite.git",
-            status.HTTP_400_BAD_REQUEST,
+            _status.HTTP_400_BAD_REQUEST,
         ),
         (
-            "f6f0acdf746c52e8e73dcc18c4de917a255e5a55dfda902fa2ea2d168b747b37",
-            "ref/head/master",
+            "e9b59c215df414f6ba45508912ce6beeb089bdab22ee3fd2e79ebdf42fe2e481",
```

Comment:
why this commit hash change?

Reply:
This is a hash that's digestible for our permissions class [ref](https://github.com/mitodl/edx-sysadmin/blob/main/edx_sysadmin/api/permissions.py#L65), Since this is generated based on the Payload we send, so the has is changed because there are changes in the payload we're sending.